scripts/__SnitchConfigServices/__SnitchConfigServices.gml

```diff
@@ -60,4 +60,8 @@
 //   N.B. HTTP warnings/failures will always be reported
 #macro SNITCH_OUTPUT_HTTP_SUCCESS  (SNITCH_RUNNING_FROM_IDE)
 
+//How many breadcrumbs to keep
```

Comment:
Link to `https://develop.sentry.dev/sdk/event-payloads/breadcrumbs/` here too please